docker-compose.yml

```diff
@@ -21,14 +21,14 @@ services:
 
   # The Grafana dashboarding server.
   grafana:
-    image: grafana/grafana:9.1.3
+    image: grafana/grafana:main
```

Comment:
Main instead of 9.3.0-beta1?

Reply:
9.3.0-beta1 doesn't currently show the panels correctly. Hence WIP. :)

When we get 9.3.0, we'll move to that and merge this PR.